### PR TITLE
Legcuffs work

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request
Allows legcuffs to function without needing to be beartraps.

# Explain why it's good for the game
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
code: Repaths handcuffs and legcuffs to have a shared parent.
code: Repaths xeno restraints to reduce confusion from the typepath.
add: Legcuffs work appropriately now, and can be found in small numbers within security vendors.
add: Cable restraints can now be adjusted via right-click to fit on wrists or ankles.
add: Prison shoe chains now require ankle restraints rather than handcuffs.
add: Prison shoes that have been chained now have an equp and unequip delay to account for the chains, proportionate to the breakout time of the item used in the chains.
/:cl:

### CODE REVIEW

File: code/game/objects/items/legcuffs.dm
Comment (harryob) [inline code/game/objects/items/legcuffs.dm:1]: it would be better to genericize this code between handcuffs/legcuffs, instead of copypasta

**Review by harryob — CHANGES_REQUESTED** [review]

File: maps/map_files/BigRed/BigRed.dmm
Comment (Nanu308) [inline maps/map_files/BigRed/BigRed.dmm:1]: mapping moment lgtm

**Review by Nanu308 — APPROVED** [review]

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by harryob — APPROVED** [review]

**Pull request merged**